src/main/java/frc/robot/Ports.java

```diff
@@ -1,15 +1,24 @@
 package frc.robot;
 
+import com.ctre.phoenix.motorcontrol.can.TalonFXConfiguration;
+
 public final class Ports {
+    public static class ExampleSubsystem {
+        public static final int MAIN = 0;
+        public static final int AUX = 0;
+        public static final boolean IS_MAIN_INVERTED = false;
+        public static final boolean IS_AUX_INVERTED = false;
+        public static final boolean IS_MAIN_SENSOR_INVERTED = false;
+        public static final boolean IS_AUX_SENSOR_INVERTED = false;
+    }
+
     public static class Shooter {
-        public static final int SOLENOID = 0;
-        public static final int MAIN_MOTOR = 0;
-        public static final boolean MAIN_INVERTED = false;
-        public static final boolean MAIN_SENSOR_PHASE = false;
+        public static final int MOTOR = 0; // Main motor port.
+        public static final TalonFXConfiguration CONFIGURATION = new TalonFXConfiguration();
```

Comment:
Should be in Constants

Reply:
Because the neutral deadband doesn't related to the ports...

Reply:
But it _is_ a configuration of the Talon FX.